src/lib/strings/time.ts

```diff
@@ -33,23 +54,26 @@ export function ago(date: number | string | Date): string {
     }
 
     if (months < 12) {
-      return `${months}mo`
+      if (MONTH_FALLBACK_LOCALES.includes(i18n.locale)) return `${months}mo`
+
+      return i18n.number(months, {
+        style: 'unit',
+        unitDisplay: 'narrow',
+        unit: 'month',
```

Comment:
```suggestion
      return i18n._(msg`${months}mo`)
```
...or, how about this?